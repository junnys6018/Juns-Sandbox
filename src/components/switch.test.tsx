import { render } from '@testing-library/react';
import Switch from './switch';

test('<Switch /> snapshot', () => {
    const { asFragment } = render(<Switch />);
    expect(asFragment()).toMatchSnapshot();
});
