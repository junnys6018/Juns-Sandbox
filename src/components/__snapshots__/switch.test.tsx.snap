// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Switch /> snapshot 1`] = `
<DocumentFragment>
  <input
    class="switch"
    type="checkbox"
  />
</DocumentFragment>
`;
